scripts/pkg/build_templates/opensearch-dashboards/deb/debian/control

```diff
@@ -8,5 +15,6 @@ Homepage: https://opensearch.org/
 
 Package: opensearch-dashboards
 Architecture: any
-Multi-Arch: foreign
-Description: OpenSearch
+Description: Open source visualization dashboards for OpenSearch
+    OpenSearch Dashboards is the visualization tool for data in OpenSearch
```

Comment:
Are both there lines required? They almost mean the same.

Reply:
Required, the 1st line is effectively the same as RPM summary, and will disable only the short line when using dpkg -l.
So yes both are required as that is how debian design their packages.